amazon-ec2-net-utils.spec

```
@@ -38,7 +38,8 @@ BuildRequires: systemd-units
 Requires: systemd-units
 %endif # systemd
 Requires: dhclient
-Provides: ec2-net-utils
+Provides: ec2-net-utils >= 1.4
```

This doesn't make a whole lot of sense in the context of Provides...

```suggestion
Provides: ec2-net-utils = %{version}-%{release}
```

---

incorporated that change, thanks